pyrk/materials/cool.py

```diff
@@ -0,0 +1,55 @@
+from ur import units
+from density_model import DensityModel
+from material import Material
+import random
+
+class Cool(Material):
```

Comment:
I meant the files with '.orig' extension. I found out where they were from
and deleted them. I am working on the material files.

On Tue, Aug 25, 2015 at 11:50 AM, Katy Huff <notifications@github.com>
wrote:

> In pyrk/materials/cool.py
> <https://github.com/pyrk/pyrk/pull/10#discussion_r37903876>:
>
> > @@ -0,0 +1,55 @@
> > +from ur import units
> > +from density_model import DensityModel
> > +from material import Material
> > +import random
> > +
> > +class Cool(Material):
>
> What do you mean by all the orig files?
> Please don't delete flibe.py. Rather, please delete "cool.py" as it is a
> duplicate of the flibe.py file. Make appropriate changes within flibe.py
> instead of inventing a duplicate.
>
> As far as safety... I'm not sure what you mean. You can "git rm" anything
> in the repository and it will remain in the history. Furthermore, this
> looks like an almost exact duplicate of flibe.py, so I'm not sure I
> understand what information you are concerned about losing.
>
> If the concern is that some simulations won't work anymore (because they
> require a material called Cool rather than a material called Flibe), then
> yes, it will be unsafe to delete this file without fixing your simulation
> input file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyrk/pyrk/pull/10/files#r37903876>.
>



-- 
-----------------------------------------
Xin WANG | 王昕
PhD Student
Thermal Hydraulics Laboratory
Nuclear Engineering Department
University of California, Berkeley
(415)408-1691